frontend/src/app/modules/hal/resources/work-package-resource.spec.ts

```diff
@@ -48,8 +48,6 @@ import {WorkPackageNotificationService} from 'core-app/modules/work_packages/not
 import {WorkPackagesActivityService} from 'core-components/wp-single-view-tabs/activity-panel/wp-activity.service';
 import {TimezoneService} from 'core-components/datetime/timezone.service';
 import {APIV3Service} from 'core-app/modules/apiv3/api-v3.service';
-import { Tab } from 'core-app/components/wp-single-view-tabs/tab/tab';
-import { TabComponent } from 'core-app/components/wp-single-view-tabs/tab/tab.component';
```

Comment:
These were just unused and introduced in my former PR. I wonder why the linter didn't complain 🤔  Anyways, now they're gone